PR_DESCRIPTION.md

```
@@ -0,0 +1,246 @@
+**Works with:**
+- **Claude Code** - Skills, Memory (CLAUDE.md), MCP
+- **Cursor** - Rules, Commands (.cursor/)
+- **GitHub Copilot** - Instructions (.github/)
+
+**All generated from `.genesis/`** - edit once, use everywhere!
+
+---
+
+## 📦 Breaking Changes (v1.x → v2.0)
+
+### Package
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)